### ISSUE TITLE
No way to set service.name on the trace provider when using the open_telemetry_collector

### ISSUE THREAD

**Issue by lfreixial** [benthosdev/benthos#1695]
Hi, Currently is not possible to set the service.name to change the name from the default **OTLPResourceNoServiceName** when using the open_telemetry_collector.

Thanks!

**Comment by mihaitodor** [comment]
Hey @lfreixial, I was able to do it by setting `service.name` in the tags section like so:

```yaml
tracer:
  open_telemetry_collector:
    http:
      - url: localhost:4318
    tags:
      service.name: benthos
```

Note: I tested this with the following Docker container:

```shell
> docker run --rm -it -e COLLECTOR_OTLP_ENABLED=true -p 4317:4317 -p 4318:4318 -p 16686:16686 jaegertracing/all-in-one:1.41
```

However, it would be nice to have a more sensible default, so I propagated the fix from #1410 in #1706.

**Comment by lfreixial** [comment]
Thats amazing, Thank you so much!
